eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -242,3 +242,9 @@ services:
             - '@ezpublish.api.repository'
         tags:
             - { name: console.command }
+
+    ezplatform.core.session.handler.native_redis:
```

Comment:
this is not really limited to redis as far as I can see. Any value to  rename service name and document it can also be used for memcached?

Reply:
The session handler is generic, the service is not.
The  $saveHandler is hardcoded in the service to "redis", see [line 250](https://github.com/ezsystems/ezpublish-kernel/pull/2245/files/8fb5fcdacf3f3a6b3a575c2f788dbaccc90758aa#diff-f72f2c40cf27099e42774203e9312f3eR250)

It was done this way to avoid yet-another-parameter. If you think we should make the service generic too, I can change this